lib/playground.dart

```diff
@@ -394,7 +394,8 @@ class Playground extends EditorUi implements GistContainer, GistController {
 
   void _handleChannelsMenuSelected(e) {
     final index = (e as CustomEvent).detail['index'] as int;
-    final channel = Channel.urlMapping.keys.toList()[index];
+    final channel = channels[index]
+        .name; // Use menu index BACK into channels array it was created from to get channel name.
```

Comment:
nit: Put the comment on the preceeding line for better formatting?

Reply:
fixed.
i admit i thought the same thing after format split it.. thanks for the prompting- lol